inst/validation/bbr-requirements.yaml

```diff
@@ -619,7 +619,7 @@ NWMD-R010:
   tests:
   - BBR-NWMD-010
 NWMD-R011:
-    description: new_model() ignores files' extensions
+  description: new_model() ignores files' extensions
```

Comment:
Could you drop this change?  Let's keep this PR restricted to the simple MPN bump.

Reply:
Is there an easy way to remove that commit?

Reply:
pinging you for call